run_unittests.py

```diff
@@ -3600,6 +3600,21 @@ def test_deterministic_dep_order(self):
                     return
         raise RuntimeError('Could not find the build rule')
 
+    def test_deterministic_rpath_order(self):
+        '''
+        Test that the rpaths are always listed in a deterministic order.
+        '''
+        if is_cygwin():
+            raise unittest.SkipTest('rpath are not used on Cygwin')
+        testdir = os.path.join(self.common_test_dir, '207 rpath order')
```

Comment:
If this test is only useful as a unit test, please put it under `test cases/unit` so that it's not also run as part of the project tests; this saves CI time.